hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdb/util/ManagedSstFileReader.java

```diff
@@ -89,6 +90,11 @@ public long getEstimatedTotalKeys() throws RocksDBException {
   }
 
   public Stream<String> getKeyStream() throws RocksDBException {
+    return getKeyStream(Optional.empty(), Optional.empty());
+  }
```

Comment:
Have we tested that this `getKeyStream()` without bounds still works?

Reply:
yeah. It is being tested with this testcase we pass Optional.empty() for both lower bound and upper bound in one of the combination since Optional.empty() is part of the testBound list. https://github.com/apache/ozone/blob/13da8647d368b5d2a17857d3535d72c772801f3e/hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdb/util/TestManagedSstFileReader.java#L127